projects/plugins/jetpack/extensions/blocks/subscriber-login/editor.scss

```diff
@@ -0,0 +1,5 @@
+/**
+ * Editor styles for Subscriber Login
+ */
+
+.wp-block-jetpack-subscriber-login { }
```

Comment:
We can probably remove the whole css file.

Reply:
That's true, removed here https://github.com/Automattic/jetpack/pull/35085